### PR TITLE
Adding notes to ablist and lgbtq files

### PR DESCRIPTION
**Pull request by wuz**
This adds some notes to terms in the LGBTQ file and one to the ablist file. Probably dependent on an answer to #60.

### CODE REVIEW

File: script/ablist.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@
 - type: simple
   source: http://ncdj.org/style-guide/
   note: If possible, describe exacly what this is.
+  # note: Assumes/implies that a person with a disability is deficient or inferior to others. When possible, specify the functional ability or its restriction.
```
Comment (wooorm) [inline script/ablist.yml]: @wuz any reason this is commented out? It looks good!

Comment (wuz) [reply script/ablist.yml]: Yep! It makes the test fail :/ 

It's the test I reference in `2.` in [this issue](https://github.com/retextjs/retext-equality/issues/60#issue-373619261) fails. Here is the code that is related to that test: https://github.com/retextjs/retext-equality/blob/master/test.js#L14

Comment (wooorm) [reply script/ablist.yml]: Ahh okay, sorry I missed that! Feel free to uncomment this line and change the test, no problem!

Comment (wuz) [reply script/ablist.yml]: Okie doke! Thanks! I'll update this soon! 😄

**Comment by wooorm** [comment]
Woohoo! Thanks @wuz! 🎉

**Comment by wuz** [comment]
Woo! Thank you! ❤️

**Pull request merged**